src/app/game-state/activity.service.ts

```diff
@@ -579,11 +579,23 @@ export class ActivityService {
       consequence: [() => {
         this.characterService.characterState.status.stamina.value -= 100;
         let oreValue = 0;
+        let builderPower = 0;
         for (let i = 0; i < 200; i++){
           oreValue = this.inventoryService.consume('ore');
         }
+        for (let follower of this.followerService.followers){
+          if (follower.job = "builder"){
+            builderPower += follower.power;
+          }
+        }
+        builderPower = Math.floor(builderPower /10);
         if (oreValue >= 10){
           this.inventoryService.addItem(this.itemRepoService.items['everlastingBrick']);
+          for (let i = 0; i < builderPower; i++){
+            this.inventoryService.addItem(this.itemRepoService.items['everlastingBrick']);
+          }
+          this.logService.addLogMessage("You and your followers made " + (1 + builderPower) + " " + this.itemRepoService.items['everlastingBrick'].name,"STANDARD","CRAFTING");;
```

Comment:
Are we off by one here? Looks like we create builderPower bricks.

Reply:
Oop, yeah that's an oversight.